### ISSUE TITLE
Test cases in examples/ do not have time dependent solutions

### ISSUE THREAD

**Issue by Keno** [SciML/ModelingToolkit.jl#1634]
The circuit examples in `examples/` are static and do not exhibit time-dependent solutions. Ideally, future improvements to structural simplification would eliminate all equations in these models. Since they're used to test various structural simplification code, it would be good to change the examples to something e.g. with an opaque, time-varying voltage source that is not static.

**Comment by ChrisRackauckas** [comment]
No, that's still an autonomous equation. The issue here is that we should have some non-autonomous equations in the tests.
